scripts/update-authors.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# 
+
+SCRIPT_DIRECTORY=`dirname $0`
+
+git log --reverse --format='%aN <%aE>' | perl -we '
+
+BEGIN {
+  %seen = (), @authors = ();
+}
+
+while (<>) {
+  next if $seen{$_};
+  next if /\@chromium.org/;
+  next if /<erik.corry\@gmail.com>/;
+  $seen{$_} = push @authors, $_;
+}
+
+END {
+  print "# Authors ordered by first contribution.\n";
+  print "\n", @authors, "\n";
+  print "# Generated by tools/update-authors.sh\n";
```

Comment:
I think the file path in the comment should be updated to `scripts/update-authors.sh` to reflect current state.

Reply:
OMG.
I forgot.